lib/xcpretty/parser.rb

```diff
@@ -217,6 +217,61 @@ module Matchers
     # @regex Captured groups
     WRITE_AUXILIARY_FILES = /^Write auxiliary files/
 
+    # @regex Captured groups
+    # $1 = directory_path
+    # $2 = target
+    CREATE_BUILD_DIRECTORY = /^CreateBuildDirectory\s(\\ |[^ ]+)\s\(in target: (.*)\)/
+
+    # @regex Captured groups
+    # $1 = directory_name
+    # $2 = target
+    MKDIR = /^MkDir\s(?:\\ |[^ ])+\/((?:\\ |[^ ])+)\s\(in target: (.*)\)/
+
+    # @regex Captured groups
+    # $1 = entitlements
+    # $2 = app_name
+    PROCESS_PRODUCT_PACKAGING = /^ProcessProductPackaging\s"([^"]*)"\s(?:\\ |[^ ])+\/((?:\\ |[^ ])+)\.xcent/
```

Comment:
1. You are right, it can also contain other types.
2. There's a pretty low chance that the entitlement's filename will contain quotes. What do you think?
3. Left as is because `xcent` is not an indicator.